src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -651,11 +651,33 @@ internal void ResetSymmetryButton()
 
     internal void OnMembraneSelected(string membrane)
     {
-        // todo: Send selected membrane to the editor script
+        editor.SetMembrane(membrane);
+    }
 
+    internal void SetSpeciesInfo(string name, MembraneType membrane, Color colour,
```

Comment:
Should this already call UpdateMembraneButtons as this is given the membrane as a parameter?